### DISCUSSION TITLE
change polarity

### DISCUSSION THREAD

**Discussion by tco95ttocs** [dlemstra/Magick.NET#803] [Help]
Hi guys,

got a problem with the polarity of my tiff. after masking with another tiff the polarity changed from min-is-white to min-is-black.
during the masking process also the compression CCIT is gone!

Under ImageMagick i can use "-define quantum:polarity=min-is-white" to change the polarity. Is there an similar command in magick.net?

thx for helping :)

**Comment by dlemstra** [comment]
You can use `image.SetDefine("quantum:polarity", "min-is-white")` to change the polarity.

**Reply by tco95ttocs** [reply]
thx for the help,

but its not setDefine its setAttribute that works 👍

**Reply by AshtonHannon** [reply]
where did you apply the SetAttribute? I used it just before the .Write call and it doesn’t seem to be working. Are there any other attributes that need to be set?
